workflows/utils/csa/README.md

```diff
@@ -14,14 +14,14 @@ git checkout develop
 ```
 
 ### 2. Set PYTHONPATH
-Here we assume you are currently inside the IMPROVE directory. This will set up `PYTHONPATH` (adds IMPROVE repo).
+Assuming you are currently inside IMPROVE directory, run the following. This will set up `PYTHONPATH` (adds IMPROVE repo).
 
 ```bash
 source setup_improve.sh
 ```
 
 ### 3. Determine the results path and run post-processing
-Here we assume the CSA results are located at `IMPROVE/workflows/utils/csa/LGBM/run.csa.small`
+Assuming the CSA results are located in `IMPROVE/workflows/utils/csa/LGBM/run.csa.small`, run the post-processing script:
```

Comment:
Don't use a path inside IMPROVE, be consistent and use the same path as mentioned above. You can use a prefix or variable to make a path example: e.g. ${EXPERIMENT_DIR}/results